### PR TITLE
SPLICE-948 Intersect Not Implemented Error on the wrong case

### PR DESCRIPTION
**Pull request by jleach4**
We do not currently support 
intersect all,
except all 
but we do support
intersect,
except.

This commit attempts to straighten that out.

### CODE REVIEW

**Comment by jyuanca** [comment]
+1

**Comment by genedavis** [comment]
+1

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SetOpOperationIT.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import java.sql.SQLException;
+import static com.splicemachine.test_tools.Rows.row;
+import static com.splicemachine.test_tools.Rows.rows;
+
+/**
+ *
+ *
+ * Test for flushing out Splice Machine handling of with clauses
+ *
+ * WITH... with_query_1 [(col_name[,...])]AS (SELECT ...),
```
Comment (dgomezferro) [inline splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SetOpOperationIT.java]: Description is wrong.

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SetOpOperationIT.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+            new TableCreator(connection)
+                    .withCreate("create table FOO2 (col1 int primary key, col2 int)")
+                    .withInsert("insert into FOO2 values(?,?)")
+                    .withRows(rows(row(1, 5), row(3, 7), row(5, 9))).create();
+
+        }
+
+    @Test
+    public void testIntercept() throws Exception {
```
Comment (dgomezferro) [inline splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SetOpOperationIT.java]: Typo, intersect.

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SetOpOperationIT.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                "select col1 from foo intersect select col1 from foo2) argh"
+        );
+        Assert.assertTrue("intersect incorrect",rs.next());
+        Assert.assertEquals("Wrong Count", 3, rs.getInt(1));
+        Assert.assertEquals("Wrong Max", 5, rs.getInt(2));
+        Assert.assertEquals("Wrong Min", 1, rs.getInt(3));
+    }
+
+    @Test(expected = SQLException.class)
+    public void testInterceptAll() throws Exception {
```
Comment (dgomezferro) [inline splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SetOpOperationIT.java]: Typo, intersect.

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SetOpOperationIT.java [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        @Rule
+        public SpliceWatcher methodWatcher = new SpliceWatcher(SCHEMA);
+
+        @BeforeClass
+        public static void createSharedTables() throws Exception {
+            TestConnection connection = spliceClassWatcher.getOrCreateConnection();
+            new TableCreator(connection)
+                    .withCreate("create table FOO (col1 int primary key, col2 int)")
+                    .withInsert("insert into FOO values(?,?)")
+                    .withRows(rows(row(1, 1), row(2, 1), row(3, 1), row(4, 1), row(5, 1))).create();
```
Comment (dgomezferro) [inline splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SetOpOperationIT.java]: We could add a couple of duplicates to make sure we remove them.

**Pull request merged**